src/main/java/cms/gov/madie/measure/services/VersionService.java

```diff
@@ -79,21 +79,31 @@ public Measure createVersion(String id, String versionType, String username, Str
     return versionQdmMeasure(versionType, username, measure, accessToken);
   }
 
+  /**
+   * @param versionType - Major or Minor Version
```

Comment:
Is Patch versioning handled somewhere else?

Reply:
Yes Patch as well, I updated both the Comments